pkg/storaged/mdraid/mdraid.jsx

```diff
@@ -194,6 +181,9 @@ function missing_bitmap(mdraid) {
 
 export function make_mdraid_page(parent, mdraid) {
     const block = client.mdraids_block[mdraid.path];
+    let running = mdraid.Running;
+    if (running === undefined)
+        running = mdraid.ActiveDevices && mdraid.ActiveDevices.length > 0;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19947-20240206-082957-2a522a56-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)